maxminddb/reader.py

```diff
@@ -152,6 +166,35 @@ def get_with_prefix_len(
             return self._resolve_data_pointer(pointer), prefix_len
         return None, prefix_len
 
+    def __iter__(self, include_aliased_nodes=False):
+        return self._generate_children(0, 0, 0, include_aliased_nodes)
+
+    def _generate_children(self, node, depth, ip_acc, include_aliased_nodes):
+        if not include_aliased_nodes and ip_acc != 0 and node == self._ipv4_start:
+            # Skip nodes aliased to IPv4
+            return
+
+        node_count = self._metadata.node_count
+        if node > node_count:
+            bits = 128 if self._metadata.ip_version == 6 else 32
+            ip_acc <<= bits - depth
+            if ip_acc <= _IPV4_MAX_NUM and bits == 128:
+                depth -= 96
+            yield ipaddress.ip_network((ip_acc, depth)), self._resolve_data_pointer(
```

Comment:
Hmm, the docs don't mention a tuple form of the parameter for ip_network(), do they?

Reply:
The parameters are passed through to the constructor and those do mention it. See, e.g., [here](https://docs.python.org/3/library/ipaddress.html#:~:text=A%20two%2Dtuple,g.%20255.255.255.0).).